src/main/java/sirius/biz/storage/util/MissingBlobObjectCheckJob.java

```diff
@@ -280,6 +282,17 @@ protected void collectAcceptedFileExtensions(Consumer<String> fileExtensionConsu
             // not relevant
         }
 
+        @Override
+        protected void computePresetFor(@Nonnull QueryString queryString,
+                                        @Nullable Object targetObject,
+                                        Map<String, Object> preset) {
+            fillPresetFromUrl(STORAGE_SPACE_PARAMETER, queryString, preset);
+            fillPresetFromUrl(INCLUDE_REPLICATION_SPACE_PARAMETER, queryString, preset);
+            fillPresetFromUrl(PARALLEL_TASKS_PARAMETER, queryString, preset);
+            fillPresetFromUrl(START_FROM_ID_PARAMETER, queryString, preset);
+            fillPresetFromUrl(CHECK_VARIANTS_PARAMETER, queryString, preset);
+        }
```

Comment:
Is this needed? Job parameters can already be prefilled if the query string in the URL contains the parameter name.

For instance, opening `/job/import-sql-user-accounts?ignoreEmpty=false` (on SQL systems) sets the "ignore empty values" parameter to false.

Reply:
Thx... reverted